website/server/libs/cron.js

```diff
@@ -347,7 +393,7 @@ export function cron (options = {}) {
 
   // Login Incentives
   user.loginIncentives++;
-  user.addNotification('LOGIN_INCENTIVE', {});
+  awardLoginIncentives(user);
```

Comment:
I've seen cases where cron exits prematurely, for example if corrupted data exists in the user's tasks. It's rare but it happens. When the only effect is that the tasks don't get adjusted and quest progress isn't made, it's not a big problem, but this feature will be popular and highly visible. Good point though about it not mattering since there's only one save location (sorry, I should have thought of that), but can we consider putting the most important cron actions together and having a save for them, then a second save for everything else? It might be more trouble than it's worth I suppose, but I'm afraid of this being another feature that gets bug reports and needs manual investigations and fixes when unexpected conditions occur. :(